.github/workflows/deploy_docs.yml

```diff
@@ -72,3 +74,43 @@ jobs:
           publish_branch: gh-pages
           destination_dir: dev
           cname: napari.org
+
+      - name: Copy permanent files locally
+        run: |
+          mkdir permanent
+          cp -R docs/docs/_build/developers permanent
+          cp docs/docs/_build/index.html permanent
+
+      - name: Upload artifacts
+        uses: actions/upload-artifact@master
+        with:
+          name: permanent_docs
+          path: permanent
+
+  copy-files:
+
+    needs: build-and-deploy
+    name: Copy permanent files
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        source_files: [index.html, developers]
+
+    steps:
+
+    - name: Download artifact
+      uses: actions/download-artifact@master
+      with:
+        name: permanent_docs
+
+    - name: Copy permanent files to napari.github.io/stable
+      uses: dmnemec/copy_file_to_another_repo_action@main
+      env:
+        API_TOKEN_GITHUB: ${{ secrets.API_TOKEN_GITHUB }}
```

Comment:
This token needs to be created on the napari.github.io repo

Reply:
I can make a secret within *this* repo (napari/docs) but not napari.github.io
Any chance the existing token used in the last deploy step: `deploy_key: ${{ secrets.ACTIONS_DEPLOY_KEY }}` could work?

Reply:
we can try, I'm not sure! As long as it has the same repo permissions I think that should work?